pages/common/elixir.md

```diff
@@ -0,0 +1,12 @@
+# elixir
+
+> Elixir programming language interpreter.
+> More information: <https://elixir-lang.org/>.
+
+- Run an Elixir file:
+
+`elixir {{path/to/file}}`
+
+- Evaluate Elixir by passing it in the command:
```

Comment:
Perhaps this will read better?
```suggestion
- Evaluate Elixir code by passing it as an argument:
```

Reply:
@sbrl he copied the example description from the `node` page

Reply:
Ah, I see. I think this makes more sense though. Maybe we should update the `node` page in a separate PR :thinking:

Reply:
I did. It feels like there are _many_ ways we could word this, and that it might be better to focus on what someone using `tldr` to go from one language to another might expect.

I'm more than happy to change this, certainly if we want to move toward a more canonical expression for these interpreters (e.g. `ruby`, `node`, `python`).

Reply:
@Schoonology please apply the suggestion from @sbrl and send another PR to edit the node page with this grammar.